plugin/trino-druid/src/test/java/io/trino/plugin/druid/TestDruidLatestConnectorSmokeTest.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.druid;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.plugin.jdbc.BaseJdbcConnectorSmokeTest;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.TestingConnectorBehavior;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import static io.trino.plugin.druid.DruidQueryRunner.createDruidQueryRunnerTpch;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestDruidLatestConnectorSmokeTest
+        extends BaseJdbcConnectorSmokeTest
+{
+    private TestingDruidServer druidServer;
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        druidServer = closeAfterClass(new TestingDruidServer("apache/druid:0.20.0"));
```

Comment:
I know the latest version is 27.0.0, but I reused the same version as TestDruidLatestConnectorTest.